arabic/learn/whattolearn/programming/programming.md

```diff
@@ -1 +1,2 @@
 ممكن أن تبدأ [بهذا](https://www.udacity.com)
+وهذا ايضا [هنا](https://www.codeschool.com)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.